exercises/collatz-conjecture/example.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+function collatzConjecture() {}
```

Comment:
As a convention, class or prototype names should be in `PascalCase`: https://github.com/airbnb/javascript#naming--PascalCase